### DISCUSSION TITLE
2-phase flow in a 3D image

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#48] [General]
Hello,

I'm Foojan, and I have a question about using a 3D image in Tutorial#2. Specifically, I'm interested in using the "Ketton" image for a two-phase flow micromodel case. Could you please advise on the modifications needed to run the case with this image?

Looking forward to hearing from you.

Best regards,
Foojan

**Comment by GeoChemFoam** [comment]
Hi,

You will need to edit the meshing and init case scripts to use a 3D input image. Use tutorial 1 as an example.



On May 3, 2024, at 11:06, Foojan-Ka ***@***.***> wrote:

﻿
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hello,

I'm Foojan, and I have a question about using a 3D image in Tutorial#2. Specifically, I'm interested in using the "Ketton" image for a two-phase flow micromodel case. Could you please advise on the modifications needed to run the case with this image?

Looking forward to hearing from you.

Best regards,
Foojan

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/48>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFUNPJMA4BJ62VISZ7LZAL5MZAVCNFSM6AAAAABHEZFDXKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGU4TQOBTG4>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Comment by Lizianne** [comment]
> Hello,
> 
> I'm Foojan, and I have a question about using a 3D image in Tutorial#2. Specifically, I'm interested in using the "Ketton" image for a two-phase flow micromodel case. Could you please advise on the modifications needed to run the case with this image?
> 
> Looking forward to hearing from you.
> 
> Best regards, Foojan

@foojan-Ka I'm also trying to run this test 2 for Ketton 3D, did you manage to do it?

**Comment by hannahmenke** [comment]
Hi Both, 

This will require substantial modification to the createmesh and raw2stl scripts. I'm afraid you will have to go through them line by line and make an effort to understand how they work then make the necessary edits. You can use tutorial 1 as a reference as they have a similar set up.
